tests/Integration/IntegrationRegistryTest.php

```diff
@@ -67,6 +67,7 @@ public function setupOnce(): void
 
         yield 'No default integrations and no user integrations' => [
             new Options([
+                'dsn' => 'http://public@example.com/sentry/1',
```

Comment:
I think it would be better to invert all these test cases so that we test only once that the `*ListenerIntegration` integrations are installed and we limit as much as possible the amount of listeners that are added to the error handler

Reply:
My reasoning was to test the 99% use case, which would be with a none empty DSN.